data2bids.m

```diff
@@ -920,7 +960,7 @@
 
 need_events_tsv       = need_events_tsv       || need_meg_json || need_eeg_json || need_ieeg_json || need_emg_json || need_exg_json || need_nirs_json || need_eyetracker_json || need_motion_json || (contains(cfg.outputfile, 'task') || ~isempty(cfg.TaskName) || ~isempty(cfg.task)) || ~isempty(cfg.events);
 need_channels_tsv     = need_channels_tsv     || need_meg_json || need_eeg_json || need_ieeg_json || need_emg_json || need_exg_json || need_nirs_json;
-need_coordsystem_json = need_coordsystem_json || need_meg_json || need_electrodes_tsv;
+need_coordsystem_json = need_coordsystem_json || need_meg_json || need_electrodes_tsv || need_nirs_json;
```

Comment:
need_nirs_json is already on line 962

Reply:
woops, that was indeed not the purpose that both branches got mixed up... I am not really done on this function, but will try to work on it later this week.

Reply:
I don't really understand what you mean... line 962 is channels.tsv while line 963 is about need_coordsystem.json, right?